src/bin/main.rs

```diff
@@ -132,6 +133,12 @@ fn make_opts() -> Options {
         "Prints the names of mismatched files that were formatted. Prints the names of \
          files that would be formated when used with `--check` mode. ",
     );
+    opts.optmulti(
+        "",
+        "config",
+        "Set options from command line. These settings take priority over .rustfmt.toml",
+        "[--config 'key1=val1,key2=val2...']",
```

Comment:
We do not need to add `--config` for the help message:

```suggestion
        "[key1=val1,key2=val2...]",
```